docs/src/main/asciidoc/redis-reference.adoc

```diff
@@ -205,14 +205,21 @@ quarkus.redis.hosts=<the redis cloud url such as redis://redis-12436.c14.us-east
 quarkus.redis.password=<the password>
 ----
 
-=== Authentication
+=== Use TLS
+
+To use TLS, you need:
+
+1. Set the `quarkus.redis.tls.enabled=true` property
+2. Make sure that your URL starts with `rediss://` (with two `s`)
+
```

Comment:
Nice. This takes care of this suggestion https://github.com/quarkusio/quarkus/issues/34761#issuecomment-1637126882 as well. Thanks

Reply:
Exactly, I was on it, so I added it.